src/util/Strings.ts

```diff
@@ -10,7 +10,8 @@ export const baseCleanup = (str: string) => str.toLowerCase().trim();
 
 export const enhancedCleanup = (str: string): string =>
     baseCleanup(str)
-        .replace(/[^a-zA-Z0-9\\s]+/g, ' ')
+        .normalize('NFKC')
+        .replace(/[^\p{L}\p{N}\s]+/gu, ' ')
```

Comment:
```suggestion
        .replace(/[^\p{L}\p{N}]+/gu, ' ')
```
the old regex unnecessarily contained `\\s` which did nothing, but `\s` leads to multiple spaces in case a char gets replaced

e.g.
`Test - 123`
old: `Test 123`
new: `Test   123`

Reply:
Nice catch! You're right, I didn't notice that.
I've updated the regex as suggested. Thanks for pointing it out!  🙌